plan/plans/where_test.go

```diff
@@ -44,7 +44,9 @@ func (t *testWhereSuit) TestWhere(c *C) {
 		Expr: &expression.BinaryOperation{
 			Op: opcode.GE,
 			L: &expression.Ident{
-				CIStr: model.NewCIStr("id"),
+				CIStr:      model.NewCIStr("id"),
+				ReferScope: 2,
+				ReferIndex: 0,
```

Comment:
0 means the index in table from fields, here is `[id, name]`, so the ReferIndex value is 0.